frontend/src/main/web/src/components/Link.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react' // eslint-disable-line
+import { flattenThemeClasses } from '../utils/styleUtils'
+import { Link as RouterLink } from 'react-router'
+
+const classes = {
+  base: {
+    c: 'C(pri)',
+    td: 'Td(n)',
+    trs: 'Trs(aeo)',
+    hover: {
+      filter: 'Brightness(.75):h'
+    },
+    focus: {
+      filter: 'Brightness(.75):f'
+    },
+    active: {
+      filter: 'Brightness(.5):a'
+    }
+  }
+}
+
+const Link = ({
```

Comment:
Needs a doc comment. Should have enough info to answer these questions:

 - Looks like this is used to add a hyperlink or an in-page navigation, is that right?
 - Are they combined to a single component so that their styling is consistent, or a different reason?